templates/vue/src/components/tyde/TydeSpaceship.vue

```diff
@@ -55,9 +56,9 @@ export default {
   },
   computed: {
     ...mapState(["lightbox"]),
-    backpackUrl() {
-      return `${wpData.vue_uri}/${BackpackIcon.split("dist")[1]}`
-    },
+    spaceshipIconUrl() {
+      return `${wpData.vue_uri}/${SpaceshipIcon.split("dist")[1]}`
+    }
```

Comment:
Please use the `getImagePath` function available in the `Helpers` class instead.